code/modules/mob/transform_procs.dm

```diff
@@ -1,7 +1,6 @@
 /mob/living/carbon/human/proc/monkeyize()
 	var/mob/H = src
 	H.dna.SetSEState(MONKEYBLOCK,1)
-	domutcheck(H, null)

```

Comment:
Just read your comment after pushing the changes, I'll amend this and have it up momentarily